docs/historias_usuario.md

```diff
@@ -0,0 +1,7 @@
+# Historias de usuario
+## [HU001] 
+Me llamo Carlos. Actualmente voy a empezar mi tercer año en Ingeniería Informática, aunque en la carrera no me ha ido muy bien, por lo que tengo muchas asignaturas pendientes.  
+Este año, al igual que el anterior, he intentado matricularme en una gran cantidad de asignaturas para reducir la cantidad de suspensas que aún tengo por aprobar. Al ser un estudiante con muchas asignaturas, siempre me encuentro con que me coinciden, debido al elevado número de materias que escojo cada año, lo que se ha convertido en un problema para mí. Me gustaría poder elegir las asignaturas de manera que, desde el primer momento, no coincidan entre ellas, evitándome problemas como tener que ir a secretaría al principio del cuatrimestre para solicitar un cambio de grupo, lo que aumenta aún más la carga de trabajo.
```

Comment:
Quizás deberías dejar claro qué es lo que hace el usuario para obtener esa información. Puedes hacer un user journey (tengo que bajarme 3 PDFs cada uno con n páginas) o añadir la información aquí. Toda la informacicón que añadas ayuda; pro ejemplo, ¿de qué universidad?

Reply:
Cuando se comenta en una línea, es para que hagáis cambios en esta línea. Si cerráis sin hacer cambios y sin haber comentado dónde se ha hecho el cambio, no se puede saber qué es lo que se ha hecho. Si has añadido los user journeys, ¿igual sería razonable enlazarlos aquí?

Reply:
[User Journey](https://github.com/JLombar/HorariosAutomatricula/pull/2/files#diff-e11f2a077252ec5746bfa517742066b0fb0e4216a28b9f3bf1daeabfdf04d486)

Reply:
Esta información no tienes que dármela a mi, sino a quien se ponga a programar, así que cuando pongas la historia de usuario en GitHub acuérdate de añadir el user journey